benchmarks/conftest.py

```diff
@@ -125,9 +138,28 @@ def get_cpu_time_for_cgroup():
     raise ValueError("Failed to find usage_usec")
 
 
+@attr.define
```

Comment:
oops one more.  how about

```suggestion
@attr.frozen
```